### ISSUE TITLE
Push images to docker HUB

### ISSUE THREAD

**Issue by loafoe** [philips-software/terraform-provider-hsdp#5]
Ability to publish image to `philipssoftware/terraform-provider-hsdp`

**Comment by loafoe** [comment]
Implemented in cdd0a334 . Now only need to figure out how to update `Dockerfile` and `README` content on Docker hub automatically.

**Comment by JeroenKnoops** [comment]
I did not find a way to do that automatically.... :(

**Comment by loafoe** [comment]
@JeroenKnoops I've updated to v2.2.0 of the ci scripts, but getting the following

```
--------------------------------------------------------------------------------------------
Update readme
--------------------------------------------------------------------------------------------
Acquiring token
Sending PATCH request
Received response code: 404
Error updating readme
no docker.io so no update
```

What changes should I make to have the readme updated automatically?

**Comment by JeroenKnoops** [comment]
Fix is made in version v2.2.1

See: https://github.com/philips-software/docker-ci-scripts/pull/39

**Comment by loafoe** [comment]
@JeroenKnoops thank you for the quick fix!
